scripts/devnet-consts.json

```diff
@@ -12,11 +12,11 @@
             "rpcUrlLocal": "http://localhost:8899",
             "rpcPort": "8899",
             "contracts": {
-                "coreEmitterAddress": "074c6d4fcaf24e4b33b3624d7c340c09e46af6fe0a86759efeb25552e46a0e2b",
```

Comment:
when submitting Emitter address into a contract, we need to pass in the Wormhole one rather than the Emitter hex right? in which case isn't it more useful to have the hexified version of the address rather than native?